code/modules/mob/living/life.dm

```diff
@@ -197,6 +198,25 @@
 		if(!remote_view && !client.adminobs)
 			reset_perspective(null)
 
+/mob/living/proc/handle_darksight(var/no_adjustment_scaling = FALSE)
```

Comment:
This is the one I'm pretty iffy on, for multiple reasons.

I admit I'm not a fan of the way it looks....

Reply:
I'd love to know precisely why so I can change it